wallet-connect-test-bench/front-end/src/Main.tsx

```diff
@@ -497,17 +498,17 @@ export default function Main(props: WalletConnectionProps) {
                                         onClick={() => {
                                             setReturnValue('');
                                             setReturnValueError('');
-                                            withJsonRpcClient(connection, (rpcClient) =>
-                                                getValue(rpcClient, useModuleSchema, readDropDown)
-                                            )
-                                                .then((value) => {
-                                                    if (value !== undefined) {
-                                                        setReturnValue(JSON.stringify(value));
-                                                    }
-                                                })
-                                                .catch((e) => {
-                                                    setReturnValueError((e as Error).message);
-                                                });
+                                            if (grpcClient) {
```

Comment:
I think rather than sliently ignoring the click if `grpcClient` is undefined, you should do the if-check where you did it for `connection` before.